### DISCUSSION TITLE
Zeek fails to validate TLS certificates

### DISCUSSION THREAD

**Discussion by sempervictus** [Security-Onion-Solutions/securityonion#8415] [Unsupported Versions]
I'm seeing a ton of messages indicating that certs are self signed or cannot be verified for things like MSFT update endpoints:
![image](https://user-images.githubusercontent.com/1331084/182024128-df53cbb2-e77a-4606-b759-ce92733c70f0.png)

IIRC there was a strange process back in the day to convert CA bundles into Bro's (at the time) preferred format.
Initial pass through docs didn't answer this for me - are there manual steps required to get TLS validation to work correctly on 2.3.140?

**Comment by presianbg** [comment]
Are you using custom CA to decrypt the SSL\TLS traffic ?
If so, please check this:
https://github.com/Security-Onion-Solutions/securityonion/discussions/3392#discussioncomment-493651

**Comment by sempervictus** [comment]
Negative, this is just public segment traffic on a default setup far as Zeek is concerned (Suricata has some customization applied through Salt parameters).
